### DISCUSSION TITLE
Power requirements

### DISCUSSION THREAD

**Discussion by mikejohnau** [PlummersSoftwareLLC/NightDriverStrip#438] [General]
Would just like to understand how we are powering some of the longer runs. I'm currently using 10A 50w power supplies per 144 LED metre strip. The strips that I use have a max pull of 43w apparently.

How would i manage a 7 metre run length?

**Comment by robertlipe** [comment]
Power injection. Run additional wires (in the channel) and or power
supplies down the line. Dont have the same strip powered in multiple places
as connecting power outputs together isn't great.

Really long distance? Run a higher voltage on the extra power line and drop
it at the last moment to the strips with buck converters.

Lots of doc in this online, often framed as wled questions. Quinn
(something) has some good YT videos on this.
